### ISSUE TITLE
Redundant and possibly harmful IsLoggedIn check

### ISSUE THREAD

**Issue by Meorawr** [wagoio/WagoAnalytics#5]
https://github.com/methodgg/WagoAnalytics/blob/b55665928dc49f451d98886806cca6a4b5093878/WagoAnalytics.lua#L76-L80

Going full nitpick mode here - I can't envisage any situation where `PLAYER_LOGIN` is delivered by the game yet `IsLoggedIn` would return false; further even if such a situation did arise I think returning here is probably wrong since `PLAYER_LOGIN` is a one-time deal and won't be fired again in the session, so you'd be left in a state where the analytics never initialized.

tldr; I think the IsLoggedIn check can just be removed.
